### PR TITLE
Updating cron in check-build and Add OSD manifest for 1.3.12

### LINKED ISSUE #3709: [RELEASE] Release version 1.3.12
## Release OpenSearch and OpenSearch Dashboards 1.3.12

I noticed that a manifest was automatically created in [manifests/1.3.12](/opensearch-project/opensearch-build/tree/main/manifests/1.3.12). Please follow the following checklist to make a release.

<details><summary>How to use this issue</summary>
<p>

## This Release Issue

This issue captures the state of the OpenSearch release, its assignee is responsible for driving the release. Please contact them or @mention them on this issue for help. There are linked issues on components of the release where individual components can be tracked.  More details are included in the Maintainers [Release owner](https://github.com/opensearch-project/opensearch-build/blob/main/MAINTAINERS.md#release-owner) section.

## Release Steps

There are several steps to the release process, these steps are completed as the whole release and components that are behind present risk to the release.  The release owner completes the tasks in this ticket, whereas component owners resolve tasks on their ticket in their repositories.

Steps have completion dates for coordinating efforts between the components of a release; components can start as soon as they are ready far in advance of a future release.

### Component List

To aid in understanding the state of the release there is a table with status indicating each component state. This is updated based on the status of the component issues.

</p>
</details>

### Preparation

- [x] Assign this issue to a release owner.
- [x] Declare a pencils down date for new features to be merged. 
- [x] **August 3rd 2023** is pencils down date for feature freeze.
- [x] Update the Campaigns section to include monitoring campaigns during this release. 
- [x] Update this issue so all `__REPLACE_RELEASE-__` placeholders have actual dates.
- [x] Document any new quality requirements or changes.
- [x] Finalize scope and feature set and update [the Public Roadmap](https://github.com/orgs/opensearch-project/projects/1). 
- [x] [Create a release issue in every component repo](https://github.com/opensearch-project/opensearch-plugins/blob/main/META.md#create-an-issue-in-all-plugin-repos) based on [component release issue template](https://github.com/opensearch-project/opensearch-build/blob/main/.github/ISSUE_TEMPLATE/component_release_template.md) and link back to this issue, update Components section with these links.
- [x] Ensure the label is created in each component repo for this new version, and the next minor release. [Create a version label](https://github.com/opensearch-project/opensearch-plugins/blob/main/META.md#create-or-update-labels-in-all-plugin-repos)
- [x] Ensure that all release issues created above are assigned to an owner in the component team.
- [x] Increase the build frequency for the this release from once a day (H 1 * * *) to once every hour (H/60 * * * *) in [jenkinsFile](https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/check-for-build.jenkinsfile).

### Campaigns

__REPLACE with OpenSearch wide initiatives to improve quality and consistency.__

### Release Branch - Ends August 3rd, 2023

- [x] Component versions are auto-incremented to 1.3.12 version.
- [x] Plugins team to ensure working CI in component repositories and merge the version increment PRs.
- [x] OpenSearch / OpenSearch-Dashboards core cut branch `<MajorVersion>.<MinorVersion>` early.
- [x] All component repos create `<MajorVersion>.<MinorVersion>` branch for the release.

### Feature Freeze - Ends August 3rd, 2023

- [x] OpenSearch / OpenSearch-Dashboards core and components teams finalize their features.
- [x] Create Jenkins workflows that run daily snapshot builds for [OpenSearch](https://build.ci.opensearch.org/job/distribution-build-opensearch/) and [OpenSearch Dashboards](https://build.ci.opensearch.org/job/distribution-build-opensearch-dashboards/).
- [x] Make pull requests to add each component to [manifests/1.3.12/opensearch-1.3.12.yml](/opensearch-project/opensearch-build/tree/main/manifests/1.3.12/opensearch-1.3.12.yml) and [manifests/1.3.12/opensearch-dashboards-1.3.12.yml](/opensearch-project/opensearch-build/tree/main/manifests/1.3.12/opensearch-dashboards-1.3.12.yml) with the corresponding checks.

### Code Complete - Ends  August 3rd, 2023

- [x] Code Complete: Make sure that the code for this specific version of the release is ready and the branch corresponding to this release has been added to this release version manifest.
- [x] Verify pull requests to add each component to [manifests/1.3.12/opensearch-1.3.12.yml](/opensearch-project/opensearch-build/tree/main/manifests/1.3.12/opensearch-1.3.12.yml) and [manifests/1.3.12/opensearch-dashboards-1.3.12.yml](/opensearch-project/opensearch-build/tree/main/manifests/1.3.12/opensearch-dashboards-1.3.12.yml) have been merged.
- [x] Gather, review and combine the release notes from components repositories.

### Release testing - Ends  August 8th, 2023

- [x] Declare a release candidate build, and provide the instructions with the release candidates for teams on testing ( August 4th, 2023).
- [x] Stop builds for this version of OpenSearch and/or OpenSearch Dashboards in order to avoid accidental commits going in unknowingly. Restart only if necessary else manually run the build workflow and declare new release candidate.
- [x] After generate the release candidates, raise PR to lock input manifest refs of both OS and OSD builds with the commit ids from distribution manifest of the release candidates.
- [x] Sanity Testing ( August 4th, 2023-  August 8th, 2023): Sanity testing and fixing of critical issues found by teams. Teams test their components within the distribution, ensuring integration, backwards compatibility, and perf tests pass.
- [x] Publish all test results in the comments of this issue.

### Performance testing validation - Ends  August 8th, 2023

- [ ] Performance tests do not show a regression.
- [ ] Longevity tests do not show any issues.

### Release - Ends August 10th, 2023

- [x] Verify [all issues labeled `v1.3.12` in all projects](https://github.com/opensearch-project/project-meta#find-labeled-issues) have been resolved.
- [x] Complete [documentation](https://github.com/opensearch-project/documentation-website) for this release.
- [x] Author [blog post](https://github.com/opensearch-project/project-website) for this release.
- [x]  August 10th, 2023 - Publish this release on [opensearch.org](https://opensearch.org/downloads.html).
- [x]  August 10th, 2023 - Publish a [blog post](https://github.com/opensearch-project/project-website) - release is launched!

### Post Release

- [x] Create [release tags](https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/release-tag/release-tag.jenkinsfile) for each component (Jenkins job name: release-tag-creation).
- [x] Replace refs in [manifests/1.3.12](/opensearch-project/opensearch-build/tree/main/manifests/1.3.12) with tags and remove checks.
- [x] If this is a major or minor version release, stop building previous patch version.
- [x] Generate distribution release notes reviewed by PM team for opensearch-build repository.
- [x] Increment version for Helm Charts [(sample PR)](https://github.com/opensearch-project/helm-charts/pull/246) for the `1.3.12` release.
- [x] Increment version for Ansible Charts [(sample PR)](https://github.com/opensearch-project/ansible-playbook/pull/50) for the `1.3.12` release.
- [x] Prepare [for next patch release](https://github.com/opensearch-project/opensearch-plugins/blob/main/META.md#increment-a-version-in-every-plugin) by incrementing patch versions for each component.
- [x] Update [this template](https://github.com/opensearch-project/opensearch-build/blob/main/.github/ISSUE_TEMPLATE/release_template.md) with any new or missed steps.
- [x] Create an issue for a retrospective, solicit feedback, and publish a summary.

### Components

__Replace with links to all component tracking issues.__

| Component | On track | Release Notes |
| --------- | -------- | ----- |
| {COMPONENT_ISSUE_LINK} | {INDICATOR}} | {STATUS} |

<details><summary>Legend</summary>
<p>

| Symbol | Meaning |
| -------- | ---------- |
| :green_circle: | On track with overall release |
| :yellow_circle: | Missed last milestone |
| :red_circle: | Missed multiple milestones |

</p>
</details>

### PR DESCRIPTION
**Pull request by Divyaasm**
### Description
Updating cron for check-build 1.3.12 to run hourly and add 1.3.12 dashboard manifest
### Issues Resolved
#3709 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: manifests/1.3.12/opensearch-dashboards-1.3.12.yml
Comment (zelinh) [inline manifests/1.3.12/opensearch-dashboards-1.3.12.yml:1]: Should this manifest be auto created by our manifest workflow once the dashboards is incremented to a new version?

Comment (Divyaasm) [reply manifests/1.3.12/opensearch-dashboards-1.3.12.yml:1]: No @zelinh. We already have the OSD incremented but the manifest file isn't created. So added it in this PR.

Comment (zelinh) [reply manifests/1.3.12/opensearch-dashboards-1.3.12.yml:1]: Looks like our manifest update job runs once a day. https://github.com/opensearch-project/opensearch-build/blob/e355eba356ad47d2c3b03e929374bd3e6b5a07dc/jenkins/manifests-update.jenkinsFile#L8
The last job runs 20hrs ago and the increment version PR was merged 19hrs ago. That's why it didn't pick up. 
https://build.ci.opensearch.org/job/manifest-update/108/

Let me trigger this job manually and see if that works.

**Review by gaiksaya — APPROVED** [review]

**Comment by gaiksaya** [comment]
Approving to avoid creating one more PR to increase cronjob frequency. Automated PRs will have once a day cronjob.
Thanks!

**Pull request merged**